### PR TITLE
Update content and table on GA360 observatory page

### PR DESCRIPTION
**Pull request by yayalu**
PDB to review content according to updated DTA communication guidelines.

### CODE REVIEW

File: content/ga360-features.md [hunk 1]
```diff
@@ -54,44 +50,35 @@ version with much lowered cost and following benefits
...
 
-Observatory has a whole-of-government contract, each new agency that joins
-unlocks more savings for everyone subscribed. The free tier of service we offer
-means that smaller agencies can save time and money. Note: Google makes Google
-Analytics free of charge up to a maximum of 10 million Hits per month per
-account.
+Observatory has a whole-of-government contract that leverages economies of scale
+to deliver better pricing for subscribers. The free tier of service that we
+offer helps smaller agencies save time and money. _Note: Google Analytics is
+free of charge up to a maximum of 10 million hits per month per account._
```
Comment (sukhrajghuman) [inline content/ga360-features.md]:
```suggestion
free of charge up to a maximum of 1 million hits per month per account._
```

File: content/ga360-features.md [hunk 2]
```diff
@@ -54,44 +50,35 @@ version with much lowered cost and following benefits
...
 
-Our contract fees are calculated on a cost recovery basis, based on the number
-of billable hits per month across all of a subscribed agencies websites.
+Our contract fees are calculated on a cost-recovery basis, based on the number
+of billable hits per month across all of the websites owned by a subscribed
```
Comment (sukhrajghuman) [inline content/ga360-features.md]: Maybe use "properties" instead of websites

File: content/ga360-features.md [hunk 3]
```diff
@@ -54,44 +50,35 @@ version with much lowered cost and following benefits
...
+agency.
```
Comment (sukhrajghuman) [inline content/ga360-features.md]: billable hits are only calculated on the properties that are upgraded to GA360.

e.g. An agency can have 100 properties, but only decides to updrage 1 of them. They are only charged for the property they've upgraded to GA360

**Review by sukhrajghuman — CHANGES_REQUESTED** [review]
few minor things to consider, other than that lgtm

**Review by patdbro — CHANGES_REQUESTED** [review]
We invoice annually (line 39) not monthly though we can negotiate other invoicing periods
line 74 "leverages the Commonwealth's economy of scale" its singular, we dont have many economies
line 75 - 76 change to "Our free tier of service helps smaller agencies ..."

File: content/ga360-features.md (hunk 3 above)
Comment (yayalu) [reply content/ga360-features.md]: Changed the above to "... across all of the properties upgraded to Google Analytics 360 by a subscribed agency." 
Is that sufficient?

**Pull request merged**